contents/docs/web-analytics/api.mdx

```diff
@@ -0,0 +1,413 @@
+---
+title: Web Analytics API
+sidebar: Docs
+showTitle: true
+availability:
+  free: partial
+  selfServe: partial
+  enterprise: partial
+---
+
+import { ProductScreenshot } from 'components/ProductScreenshot'
+
+The Web Analytics API allows you to programmatically access your web analytics data outside of PostHog. This is useful for building custom dashboards, integrating with other tools, or creating your own analytics applications.
+
+> **Note:** The Web Analytics API is currently in beta. Please [contact support](https://app.posthog.com/#panel=support%3Asupport%3Aweb_analytics%3Alow%3Atrue) to enable it for your team or [sign up to get notified when it's released](https://app.posthog.com/settings/user-feature-previews#web-analytics-api).
+
+## Prerequisites
+
+To use the Web Analytics API, you need:
+
+1. A [personal API key](https://app.posthog.com/settings/user-api-keys) with `query:read` permissions
+2. Your project ID (found in [project settings](https://app.posthog.com/settings/project#variables))
+
+## Endpoints
+
+The Web Analytics API provides two main endpoints:
+
+- **Overview** - Get aggregated metrics like total visitors, views, sessions, bounce rate, and session duration for a date range
+- **Breakdown** - Get metrics broken down by properties like page paths, countries, browsers, device types, and more
+
+> **Full API documentation:** For complete endpoint specifications, see the [Web Analytics API reference](/api/docs/web-analytics).
```

Comment:
I would suggest updating the `/api/docs/web-analytics` to link here as the guide they can follow vs references.